alot/crypto.py

```diff
@@ -49,7 +19,7 @@ def RFC3156_micalg_from_algo(hash_algo):
     :rtype: str
     """
     # hash_algo will be something like SHA256, but we need pgp-sha256.
-    hash_algo = _hash_algo_name(hash_algo)
+    hash_algo = gpg._gpgme.gpgme_hash_algo_name(hash_algo)
```

Comment:
Any reason for not having a wrapper? (can we request one?)

The leading underscore by convention denotes private in python, so if this is going to go in as-is I'd at least want a comment explaining the situation (perhaps linking to @teythoon's comment). Ideally upstream would expose this and then we could just call the wrapper.